acra/guides/djangoproject-short-tutorial.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Integration example: DjangoProject - short tutorial"
+bookCollapseSection: true
+---
+
+# Integrating Acra with DjangoProject
```

Comment:
Do we want to make some reference on our engineering demos repo which also contains some django tutorials?

Reply:
agree, added